bitshuffle/bitshuffle.py

```diff
@@ -380,6 +380,7 @@ def set_defaults(args):
 
     # open args.input and args.output so they are file handles
     if not isinstance(args.input, file_type):
+        stderr.write("opening input")
         try:
```

Comment:
While I should probably have remembered to remove this, are we certain we want debug messages to fail our build tests? See https://ci.appveyor.com/project/charlesdaniels/bitshuffle/build/38 for context.